app/main.py

```diff
@@ -19,11 +21,25 @@ def index():
 
 def generate_badges():
     os.system(SCRIPT)
+
+
+def empty_directory():
+    for file in os.listdir(BADGES_FOLDER):
+        file_path = os.path.join(BADGES_FOLDER, file)
+        try:
+            if os.path.isfile(file_path):
+                os.unlink(file_path)
+            elif os.path.isdir(file_path): shutil.rmtree(file_path)
+        except Exception:
+            pass
+
+
 @app.route('/upload', methods=['POST'])
 def upload():
-
+    empty_directory()
```

Comment:
If we have multiple uploads at the same time, could they block eachother?

Reply:
No. In case we create multiple upload feature, this will still work seamlessly without fail :)